test/utils_for_tests.py

```diff
@@ -469,6 +470,15 @@ def robot(self):
         """
         return self.world.groups['robot']
 
+    def dye_group(self, group_name, rgba: list):
+        req = DyeGroupRequest()
+        req.group_name = group_name
+        req.color.r = rgba[0]
+        req.color.g = rgba[1]
+        req.color.b = rgba[2]
+        req.color.a = rgba[3]
+        return self.dye_group_srv(req)
```

Comment:
add an expected error code parameter, similar to the other service calls and add tests

Reply:
then assert the error code and instead of returning it

Reply:
Please also add a similar function to the python interface, but without the expected errorcode parameter obviously